connectors/sources/sharepoint_server.py

```diff
@@ -462,54 +489,65 @@ def get_default_configuration(cls):
             dictionary: Default configuration.
         """
         return {
+            "authentication":{
```

Comment:
You need "display": "dropdown" here to make it an actual dropdown - now it looks like a textfield

Reply:
Fixed in next commit.